docs/import-data/import-touristic-data-systems.rst

```diff
@@ -268,6 +269,34 @@ If you use an url that filters a unique category, you can change its name. Examp
             'type1': "GeotrekTypeName",
         }
 
+.. _import-from-openstreetmap:
+
+Import from OpenStreetMap
+==========================
+
+Import information desks
+-------------------------
+
+To import information desks from OpenStreetMap, edit the ``/opt/geotrek-admin/var/conf/parsers.py`` file with the following content:
+
+::
+
+    from geotrek.tourism.parsers import InformationDeskOpenStreetMapParser
+
+    class MaisonDuParcParser(InformationDeskOpenStreetMapParser):
+        provider = "OpenStreetMap"
+        tags = {"amenity": "ranger_station"}
+        type = "Maisons du parc"
+
+Then set up appropriate values:
+
+* ``tags`` to filter the objects imported from OpenStreetMap (see `MapFeatures <https://wiki.openstreetmap.org/wiki/Map_features/>`_  to get a list of existing tags)
+* ``type`` to specify the Geotrek type for imported objects
+* See the `geotrek/tourism/parsers.py/ <https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/tourism/parsers.py/>`_  file for details about parsers
+
+The parsed objects will be those contained in the ``setting.SPATIAL_EXTENT`` bounding box.
```

Comment:
You can override `get_bbox_str`method to define your own custom bounding box per parser.